### DISCUSSION TITLE
[2021.08.30] 입양 시각 구하기(1)

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#78] [SQL]
[입양 시각 구하기(1)](https://programmers.co.kr/learn/courses/30/lessons/59412)

**Comment by pyunji** [comment]
[입양 시각 구하기(1) solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59412.sql)

**Comment by hyk339** [comment]
[입양 시각 구하기(1) solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59412.sql)

**Reply by pyunji** [reply]
oracle이 아니라 mysql로 푸신 것 같습니다 👀

**Comment by chaeyeon4u** [comment]
[입양 시각 구하기(1) solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59412.sql)

**Reply by chaeyeon4u** [reply]
https://gent.tistory.com/331

**Comment by leechang4** [comment]
[입양 시각 구하기(1)](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059412.sql)
